docs/source/Launcher/index.rst

```diff
@@ -53,18 +55,23 @@ There will be editing of scripts, extracting an icon from the MyM-Launcher.jar a
 * Commands using the home userland work directory paths will be instructed.
 
 1. Install Oracle JDK from Command Line 
+  * This installs Java 8 for the entire system. 
+  * This Java is recommended but other variations of Java will work.
+  
   .. code-block:: none
   
     sudo add-apt-repository ppa:webupd8team/java
     sudo apt-get update
     sudo apt-get install oracle-java8-installer
   
-  * Make sure to agree to the terms and wait for it to finish installing
+  * Make sure to agree to the terms and wait for it to finish installing. This can take some time. 
 
 2. Download and create **custom path** to all files
-  * Download the MyM Launcher directly from `here <https://mineyourmind.net/#dl_jar>`_
-  * *Please manage your own files and folders at your desire*
-  * Move or ``copy mv`` or ``cp``. the ``MyM-Launcher.jar`` to a folder designated
+  * Download The Official free MyM Launcher directly from `here <https://mineyourmind.net/#dl_jar>`_
+  * ***Please manage your own files and folders at your desire***
```

Comment:
yeah that doesnt make sense. 

cp or mv (copy or move) might be better to explain those commands.